content/en/_index.html

```diff
@@ -93,6 +93,25 @@ <h5 class="card-title text-white section-head">Dashboard</h5>
     </div>
     <br />
     <div class="card-deck">
+      <div class="card border-primary-dark">
+         <a href="/docs/components/trainer/overview/" target="_blank" rel="noopener" >
+           <img
+             src="/docs/images/logos/kubeflow-trainer.png"
+             class="card-img-top"
+             draggable="false"
+             style="padding: 2rem;"
+             alt="Kubeflow Trainer logo"
+           />
```

Comment:
@thesuperzapper Do you know what should we adjust to modify padding for the Kubeflow Trainer Icon ?
![Screenshot 2025-02-06 at 14 58 53](https://github.com/user-attachments/assets/1f77b0bc-3e37-4211-8c46-63d5d67b92ae)